pkg/cableengine/cableengine.go

```diff
@@ -168,6 +168,11 @@ func (i *engine) getEndpointInfo(cableName string, connections *[]v1.Connection)
 }
 
 func (i *engine) RemoveCable(endpoint types.SubmarinerEndpoint) error {
+	if endpoint.Spec.ClusterID == i.localCluster.ID {
+		klog.V(log.DEBUG).Infof("Not removing cable for local cluster")
```

Comment:
```suggestion
		klog.V(log.DEBUG).Infof("Cables are not added/removed for local cluster, skipping removal")
```

To incorporate your explanation?